package.json

```diff
@@ -2,18 +2,25 @@
   "name": "@geoblocks/sources",
   "version": "0.1.3",
   "description": "Geoblocks sources",
-  "module": "src/index.js",
+  "main": "dist/sources.js",
```

Comment:
You are not building a CJS version. Is it OK to have a `main` attribute?

Reply:
Unfortunately TypeScript doesn't take into account the value of `"module"`, so if `"main"` is removed, I have to import like this:
```
import { Swisstopo as SwisstopoSource } from '@geoblocks/sources/dist/sources';
```
instead of
```
import { Swisstopo as SwisstopoSource } from '@geoblocks/sources';
```

(which is of course also a possibility, I can change it if you want, but I thought since anyway we don't have a CJS alternative to put as `"main"` it was easier like this)